docs/content/preview/explore/observability/metrics-overview.md

```diff
@@ -0,0 +1,120 @@
+---
+title: YugabyteDB metrics
+headerTitle: YugabyteDB metrics
+linkTitle: Metrics overview
+description: Learn about YugabyteDB database metrics
+menu:
+  preview:
+    identifier: metrics-overview
+    parent: explore-observability
+    weight: 200
+type: docs
+---
+
+YugabyteDB provides thousands of metrics. Use the information on this page to better understand your options for selecting and consuming the metrics relevant to your situation. On this page, you'll learn about the available metrics, their syntax and semantics, and how to view them for a specific use case.
+
+## Export YugabyteDB metrics
+
+Each node in a YugabyteDB cluster exports metrics through JSON and Prometheus Exposition Format API endpoint. You can aggregate the metrics from each node to get a cluster-wide view. YugabyteDB Anywhere (YBA) and YugabyteDB Managed (YBM) aggregate a sub-set of relevant metrics from each node and provide an aggregated cluster view.
+
+### Export by logical architecture components
+
+| Component | Description | JSON API endpoints | Prometheus API endpoint |
+| :-------- | :---------- | :----------------- | :---------------------------------------- |
+| YSQL | YSQL statement and connection metrics. <br/> Starting point for all YSQL query-related heuristics. | | :13000/metrics | :13000/prometheus-metrics |
+| YCQL | YCQL statement and connection metrics. <br/> Starting point for all YCQL query-related heuristics. | :12000/metrics | :12000/prometheus-metrics |
+| Master | | :7000/metrics | :7000/threadz | :7000/prometheus-metrics |
+| TServer | | :9000/metrics | :9000/threadz | :9000/prometheus-metrics |
```

Comment:
Description:
Yugabyte Tablet Server service, Postgres and storage layer metrics

Value:
Debug level metrics to troubleshoot system and  storage issues